xun/tests/helpers.py

```diff
@@ -169,7 +169,9 @@ def sample_sin(offset, sample_count, step_size):
 
     blueprint = sample_sin.blueprint(offset, sample_count, step_size)
     expected = [
-        sin(radians(i / step_size)) + offset for i in range(sample_count)
+        sin(radians(i * step_size)) + offset for i in range(sample_count)
     ]
+    print(blueprint, " = blueprint")
+    print(expected, " = expected")
```

Comment:
We don't need print statement in tests. Please, remove.

Reply:
The prints have been removed.